spec/consensus/proposer-based-timestamp/tla/TendermintPBT_002_draft.tla

```diff
@@ -684,10 +790,8 @@ MessageProcessing(p) ==
  * A system transition. In this specificatiom, the system may eventually deadlock,
  * e.g., when all processes decide. This is expected behavior, as we focus on safety.
  *)
-Next ==
+Next == 
   \/ AdvanceRealTime
```

Comment:
Follow up on the deadlock, if I disable the deadlock check, in TLC actions I only see `AdvanceRealTime` (in addition to `Init` and `Next`). And a simple run (2 correct processes) never finishes.

Reply:
Ah actually it did finish.